utils.py

```diff
@@ -226,63 +225,12 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
 
     label_map = {label : i for i, label in enumerate(label_list)}
 
-    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id) for example in examples]
+    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id, sep_token_extra) for example in examples]
 
     with Pool(process_count) as p:
         features = list(tqdm(p.imap(convert_example_to_feature, examples, chunksize=100), total=len(examples)))
 
     return features_length - len(input_ids)
-    if pad_on_left:
```

Comment:
This whole thing looks like a bad merge or copy/paste

Reply:
Yeah, it looks like the code from `convert_example_to_feature` function somehow got copied into the `convert_examples_to_features` function (as a duplicate to boot). Thanks, I'll merge this.